components/checkbox.json

```diff
@@ -0,0 +1,48 @@
+{
+  "checkbox": {
+    "checked":{
+		"#normal":{
+			"background": "@theme-control-active-normal",
+			"text": "@theme-text-primary-normal",
+			"border": "@theme-control-active-normal"
+		},
```

Comment:
What colour is the actual check icon? There should be a token for that.

Reply:
The icon color is added. 👍